ui/app/AppLayouts/Wallet/views/AssetsDetailView.qml

```diff
@@ -62,6 +62,14 @@ Item {
 
         property bool forceRefreshBalanceStore: false
         readonly property var splitAddresses: root.networkFilters.split(":")
+
+        readonly property SortFilterProxyModel enabledNetworksModel: SortFilterProxyModel {
+            sourceModel: root.allNetworksModel
+            filters: ValueFilter {
+                roleName: "isEnabled"
+                value: true
+            }
+        }
```

Comment:
Ideally, this filtering should be done upper in the hierarchy. `AssetsDetailsView` should just display, without any additional preparation. Basically the same thing as you did for `AssetsDetailsHeader`.